Writing-state-migrations.md

```diff
@@ -78,10 +75,11 @@ If you find new test files where changes needed to be required, try updating the
 - Create a new exploration, make some changes and save them.
 - Checkout the feature branch which contains state migration.
 - Go to 0.0.0.0:8000 and flush existing Memcache from the Memcache tab.
-- Go to the Misc tab of admin page and flush cache.
-- Go to the Jobs tab of admin page.
-- Run ExplorationMigrationJobManager and wait for the job to get completed.
+- Go to admin page and assign yourself the role of `release coordinator`.
+- Go to the Misc tab of release-coordinator page and flush cache.
+- Run AuditExplorationMigrationJob and wait for the job to get completed. This job will not make any changes to the exploration as this is simply an audit job. We run this job before the actual migration job so that in case anything fails we do not make changes to the datastore.
+- Run MigrateExplorationJob and wait for the job to get completed. This will make changes to exploration.
```

Comment:
```suggestion
- Go to the admin page and assign yourself the role of "release coordinator".
- Go to the Misc tab of /release-coordinator page and flush the cache.
- Run `AuditExplorationMigrationJob` and wait for the job to get completed. This job will not make any changes to the exploration as this is simply an audit job. We run this job before the actual migration job so that in case anything fails we do not make changes to the datastore.
- Run `MigrateExplorationJob` and wait for the job to get completed. This will make changes to exploration.
```

Reply:
Updated